source/declarations.tex

```diff
@@ -1771,9 +1771,11 @@
 
 \pnum
 The type of a \grammarterm{parameter-declaration} of a
-function declaration\iref{dcl.fct},
-\grammarterm{lambda-expression}\iref{expr.prim.lambda}, or
-\grammarterm{template-parameter}\iref{temp.param}
+\begin{itemize}
+  \item function declaration\iref{dcl.fct},
+  \item \grammarterm{lambda-expression}\iref{expr.prim.lambda},
```

Comment:
"or" missing at the end.

Reply:
ok, I wasn't sure whether to add, I will refine soon.